app/routers/channels.py

```diff
@@ -0,0 +1,93 @@
+from fastapi import APIRouter
+import schemas
+
+
+channels_router = APIRouter(
+    prefix = "/channels",
+    tags = ["channels"]
+)
+
+
+@channels_router.get("/")
+async def get_list():
+    return {'status':'OK'}
+
+
+@channels_router.post("/")
+async def create_channel(info: schemas.Channel):
+    return {'status':'OK'}
+
+
+@channels_router.get("/{channel_id}")
+async def get_info_channel(info: schemas.Channel):

```

Comment:
В пути объявлен параметр пути channel_id, но в функцию он не передаётся. 
Следует добавить его следующим образом
```suggestion
async def get_info_channel(channel_id: int, info: schemas.Channel):

```